platform/darwin/src/MGLMapSnapshotter.mm

```diff
@@ -726,9 +726,23 @@ - (void)configureWithOptions:(MGLMapSnapshotOptions *)options {
 
     // Create the snapshotter
     mbgl::optional<std::string> localFontFamilyName = config.localFontFamilyName ? mbgl::optional<std::string>(std::string(config.localFontFamilyName.UTF8String)) : mbgl::nullopt;
+    mbgl::GlyphsRasterizationOptions glyphsRasterizationOptions;
```

Comment:
If `MGLIdeographicFontFamilyName` is provided and it is set to `NO`, and `MGLGlyphsRasterizationMode` is set to something other than `MGLNoGlyphsRasterizedLocally`, maybe we should print error and fallback to `NoGlyphsRasterizedLocally` option.

Maybe worth adding test for that combination?

Reply:
Definitely.